app/assets/javascripts/containers/MessagesContainer.coffee

```diff
@@ -0,0 +1,9 @@
+mapStateToProps = (state) ->
```

Comment:
Goed punt, redelijk nutteloos deze functies.

Blijkt dat `mapDispatchToProps` een optionele parameter is. Die kan dus helemaal weg